lib/browser.js

```diff
@@ -117,7 +119,17 @@ class Browser {
   }
 
   async _launch (preview) {
+    // if creating binaries with pkg, we use bundled chromium instances stored in
+    // the `chromium` dir instead of chromium installed as part of the puppeteer package
+    const chromiumExecutablePath = (isPkg
+      ? puppeteer.executablePath().replace(
+          /^.*?\/node_modules\/puppeteer\/\.local-chromium/,
```

Comment:
I think this is why it's not working because `puppeteer.executablePath()` will be `D:\snapshot\asciidoctor-web-pdf\node_modules\puppeteer\.local-chromium\win64-818858\chrome-win\chrome.exe` on Windows.

Since we are using forward slashes in the regular expression the `node_modules\puppeteer\.local-chromium` won't be replaced by `chromium`.

Reply:
Oh that seems likely, good job. I must have tested on Windows before this point in time.

Reply:
I change the logic a bit to keep everything after the `.local-chromium` directory. Not very concise but it should be reliable. WE could also use `split` with a fixed number since the directory structure seems to be stable on every platform (see: https://github.com/vercel/pkg/issues/204#issuecomment-536323464) 

Now that's it's working on every platform, I will make some more tests.

I think I will also add support for `PUPPETEER_EXECUTABLE_PATH`. I believe it's harmless (i.e., if the environment variable is undefined it won't change a thing) and it can be useful to experiment.